src/main/kotlin/no/skatteetaten/aurora/boober/service/DeployService.kt

```diff
@@ -87,22 +87,21 @@ class DeployService(
 
     private fun prepareDeployEnvironment(environment: AuroraDeployEnvironment, projectExist: Boolean): List<OpenShiftResponse> {
 
-        val affiliation = environment.affiliation
         val namespace = environment.namespace
 
-        val createNamespaceResponse = openShiftObjectGenerator.generateProjectRequest(environment).let {
-            openShiftClient.createOpenShiftCommand(namespace, it, projectExist)
-        }.let { openShiftClient.performOpenShiftCommand(namespace, it) }
+        val projectRequestResponse = openShiftObjectGenerator.generateProjectRequest(environment)
```

Comment:
Should we consider putting the generated resources in a collection and running a single createOpenShiftCommand && performOpenShift command on them?

Reply:
Yes, we should probably do that. If I recollect correctly this was not possible before your prepareDeployEnvironent refactoring. But I think it is now.

Reply:
Agree